code/__HELPERS/unsorted.dm

```diff
@@ -508,14 +508,16 @@ Recursively gets all contents of contents and returns them all in a list.
 
 recursive_depth is useful if you only want a turf and everything on it (recursive_depth=1)
 */
-/atom/proc/GetAllContents(list/output=list(), recursive_depth=INFINITY)
+/atom/proc/GetAllContents(list/output=list(), recursive_depth=199, _current_depth=0)	// 200 proc calls in and we break shit
```

Comment:
this should be a define MAX_PROC_DEPTH

Reply:
Also, I'm not sure why you added a new arg when decreasing recusive_depth would be better, faster, and cheaper memory wise.